src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -45,8 +52,14 @@ std::mutex err_print_lock;
 
 using namespace dnnl::impl::utils;
 using namespace dnnl::impl::cpu;
+
+#if !defined(OPENVINO_ARCH_ARM64)
```

Comment:
Thanks Alexandra for the comment! I was using your suggested approach in the first place. I revised it because there was it lot of compiling errors on RISC-V platform in CI tests. Before this PR, RISC-V goes the x64 way, now it is excluded from it in someplaces. So I suggest the following way to improve readability like you suggested, and also maintain some compatibility. Further discussions are always welcomed! Thanks!
`#if defined(OPENVINO_ARCH_ARM64)`
`...`
`else`
`...`
`#endif`